.projen/tasks.json

```diff
@@ -98,35 +98,27 @@
     },
     "upgrade-dependencies": {
       "name": "upgrade-dependencies",
-      "description": "upgrade dependencies",
```

Comment:
Looks like you downgraded projen for some reason... Are you running `npx projen` or globally installed?

Reply:
Whoops. Usually I yarn install and then 'yarn projen' to keep it local. 'npx projen' has given me some issues in the past. Will fix